### ISSUE TITLE
SC 2.5.3 - Initial adjustment - Top right search pattern

### ISSUE THREAD

**Issue by duboisp** [wet-boew/GCWeb#1923]
The top right search component need to be adapted for conformity with WCAG SC 2.5.3 Label in Name.

It FAIL [F96](https://www.w3.org/WAI/WCAG21/Techniques/failures/F96)

Requirement, the following must match their accessible name

* Label field
* Placeholder attribute
* Search button.

The following will be completed
* Working example where it used in all Canada.ca pages
* Working example where it use a custom top right search, ex. CRA
   * Include the documentation of the additional template page variable that will be added
* An index/documentation pages in the folder “sites/search” with showing an example and the expected markup.

**Comment by duboisp** [comment]
After review, this is not an issue. The label for the text input are not linked with the label on the submit button. And the submit button don’t have any linkage to another label so the Failure 96 is not applicable here.

We can cancel this request now.
